src/token_contract/src/test/utils.nr

```diff
@@ -271,6 +271,7 @@ pub unconstrained fn get_private_balance(
     amt
 }
 
+// TODO: I think we don't need this anymore.
```

Comment:
in a way, it's more "unit test" like to just add token notes instead of "minting to private", but idk if we should need this method